Build/scripts/pgo/pogo_training.ps1

```diff
@@ -27,9 +23,18 @@ param (
 
     # force callers to specify this in case of future use
     [Parameter(Mandatory=$True)]
-    [string]$flavor = ""
+    [string]$flavor,
+
+    [string]$vcinstallroot = ${env:ProgramFiles(x86)},
```

Comment:
I wanted to remove these in favor of a prebuild step. We can talk about it in person if you want